editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -127,7 +127,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`859`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`907`)
```

Comment:
do you know what caused the render count increase? first glance looks like a regressed memoization

Reply:
after some discussion we agreed that this is OK, since we're still iterating on the data picker anyway